lib/runner.js

```diff
@@ -873,8 +833,8 @@ Runner.prototype.run = function(fn) {
   if (this._delay) {
     // for reporters, I guess.
     // might be nice to debounce some dots while we wait.
-    this.emit('waiting', rootSuite);
-    rootSuite.once('run', start);
+    this.emit(constants.EVENT_RUN_DELAY, rootSuite);
+    rootSuite.once(Suite.constants.EVENT_ROOT_SUITE_RUN, start);
```

Comment:
Still think these two would work better as `EVENT_RUN_DELAY_BEGIN` and `EVENT_RUN_DELAY_END`...